tools/build_pytorch_libs.py

```diff
@@ -126,6 +126,7 @@ def build_pytorch(
         not check_negative_env_flag("USE_CUDA")
         and not check_negative_env_flag("USE_NCCL")
         and not check_env_flag("USE_SYSTEM_NCCL")
+        and not check_env_flag("USE_SYSTEM_LIBS")
```

Comment:
Would't the proper fix be USE_SYSTEM_LIBS should override USE_SYSTEM_NCCL?

Reply:
USE_SYSTEM_LIBS is false by default, and we can still set USE_SYSTEM_NCCL to true. When they're both false we need to checkout nccl.

Reply:
True, I guess my question is:
Does USE_SYSTEM_LIBS imply USE_SYSTEM_NCCL?
I am saying is there any valid configuration where:
`USE_SYSTEM_LIBS` is True and `USE_SYSTEM_NCCL` should be false?

Reply:
I see. It seems `USE_SYSTEM_LIBS` only overwrites `USE_SYSTEM_NCCL` when it's true. by default it's false and this do not affect the value of `USE_SYSTEM_NCCL`